### ISSUE TITLE
404 Not Found comment

### ISSUE THREAD

**Issue by yegor256** [yegor256/wring#72]
I keep getting this 10-50 times a day in the log:

```
java.lang.AssertionError: HTTP response status is not equal to 200:
404 Not Found [https://api.github.com/repos/Harvest-foundation/core/issues/185/comments]
Server: GitHub.com
Date: Mon, 15 Oct 2018 11:15:44 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4991
X-RateLimit-Reset: 1539605620
X-OAuth-Scopes: notifications
X-Accepted-OAuth-Scopes: 
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-GitHub-Request-Id: BDBC:740F:15106FC:2CA2220:5BC476E0

{"message":"Not Found","documentation_url":"https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue"}
Expected: HTTP response with status 200
     but: was <404 Not Found [https://api.github.com/repos/Harvest-foundation/core/issues/185/comments]
Server: GitHub.com
Date: Mon, 15 Oct 2018 11:15:44 GMT
Content-Type: application/json; charset=utf-8
Transfer-Encoding: chunked
Status: 404 Not Found
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4991
X-RateLimit-Reset: 1539605620
X-OAuth-Scopes: notifications
X-Accepted-OAuth-Scopes: 
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-GitHub-Request-Id: BDBC:740F:15106FC:2CA2220:5BC476E0

{"message":"Not Found","documentation_url":"https://developer.github.com/v3/issues/comments/#list-comments-on-an-issue"}>
    at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
    at com.jcabi.http.response.RestResponse.assertStatus(RestResponse.java:111)
    at com.jcabi.github.RtValuePagination$Items.fetch(RtValuePagination.java:193)
    at com.jcabi.github.RtValuePagination$Items.hasNext(RtValuePagination.java:179)
    at com.jcabi.github.Smarts$1.hasNext(Smarts.java:87)
    at io.wring.agents.github.BoIssue.text(BoIssue.java:126)
    at io.wring.agents.github.BoIssue.push(BoIssue.java:79)
    at io.wring.agents.github.Subject.push(Subject.java:130)
    at io.wring.agents.github.AgGithub.push(AgGithub.java:172)
    at io.wring.agents.github.AgGithub.push(AgGithub.java:110)
    at io.wring.agents.JsonAgent.push(JsonAgent.java:97)
    at io.wring.agents.Exec.log(Exec.java:188)
    at io.wring.agents.Exec.body(Exec.java:161)
    at io.wring.agents.Exec.lambda$run$0(Exec.java:98)
    at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:84)
    at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
    at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
    at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
    at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
    at io.wring.agents.Exec.run(Exec.java:142)
    at io.wring.agents.Cycle.lambda$exec$2(Cycle.java:96)
    at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:92)
    at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
    at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
    at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
    at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
    at io.wring.agents.Cycle.exec(Cycle.java:101)
    at io.wring.agents.Cycle.exec(Cycle.java:54)
    at org.cactoos.func.FuncOf.lambda$new$3(FuncOf.java:88)
    at org.cactoos.func.FuncOf.apply(FuncOf.java:104)
    at org.cactoos.func.FuncWithFallback.apply(FuncWithFallback.java:84)
    at org.cactoos.func.UncheckedFunc.lambda$apply$0(UncheckedFunc.java:58)
    at org.cactoos.scalar.IoCheckedScalar.value(IoCheckedScalar.java:70)
    at org.cactoos.scalar.UncheckedScalar.value(UncheckedScalar.java:58)
    at org.cactoos.func.UncheckedFunc.apply(UncheckedFunc.java:59)
    at org.cactoos.func.RunnableOf.run(RunnableOf.java:89)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.run(FutureTask.java:266)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
    at com.jcabi.log.VerboseThreads$Wrap.run(VerboseThreads.java:220)
    at java.lang.Thread.run(Thread.java:748)
```

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#72](https://github.com/yegor256/wring/issues/72) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Comment by 0crat** [comment]
The job [#72](https://github.com/yegor256/wring/issues/72) assigned to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), here is [why](https://www.0crat.com/footprint/C7FCB1EQN/2890557112); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be a monetary reward for this job

**Comment by paulodamaso** [comment]
@g4s8 Would you please check if wring have access to `Harvest-foundation/core` repository? Looks like a permission problem to me.

**Comment by g4s8** [comment]
@paulodamaso no it doesn't

**Comment by g4s8** [comment]
@paulodamaso why? Actually it was not intended to add wring to this repo.

**Comment by paulodamaso** [comment]
@g4s8 Right, I thought that you was trying to use wring, sorry.

**Comment by paulodamaso** [comment]
@carlosmiranda Looks like we have a problem with user authentication; maybe user token is not valid, and its not used for accessing public repos, so it just complains when accessing private repos. To solve this issue I need to know how wring must behave when this problem happens.

**Comment by 0crat** [comment]
@paulodamaso[/z](https://www.0crat.com/u/paulodamaso) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by paulodamaso** [comment]
@0crat wait for #75 review

**Comment by 0crat** [comment]
> @0crat wait for #75 review ([here](https://github.com/yegor256/wring/issues/72#issuecomment-432379535))

@paulodamaso The impediment for [#72](https://github.com/yegor256/wring/issues/72) was registered successfully by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#76](https://github.com/yegor256/wring/issues/76), [#77](https://github.com/yegor256/wring/issues/77), [#78](https://github.com/yegor256/wring/issues/78) are still not solved.

**Comment by paulodamaso** [comment]
@yegor256 done and merged, please close

**Comment by paulodamaso** [comment]
@yegor256 ping

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#76](https://github.com/yegor256/wring/issues/76), [#78](https://github.com/yegor256/wring/issues/78), [#82](https://github.com/yegor256/wring/issues/82) are still not solved; solved: [#77](https://github.com/yegor256/wring/issues/77).

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#76](https://github.com/yegor256/wring/issues/76), [#82](https://github.com/yegor256/wring/issues/82), [#83](https://github.com/yegor256/wring/issues/83) are still not solved; solved: [#77](https://github.com/yegor256/wring/issues/77), [#78](https://github.com/yegor256/wring/issues/78).

**Comment by 0pdd** [comment]
@yegor256 4 puzzles [#82](https://github.com/yegor256/wring/issues/82), [#83](https://github.com/yegor256/wring/issues/83), [#85](https://github.com/yegor256/wring/issues/85), [#86](https://github.com/yegor256/wring/issues/86) are still not solved; solved: [#76](https://github.com/yegor256/wring/issues/76), [#77](https://github.com/yegor256/wring/issues/77), [#78](https://github.com/yegor256/wring/issues/78).

**Comment by paulodamaso** [comment]
@yegor256 ping

**Comment by paulodamaso** [comment]
@yegor256 please close this issue, it was merged almost one month ago

**Comment by yegor256** [comment]
@paulodamaso thanks!

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @paulodamaso[/z](https://www.0crat.com/u/paulodamaso), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#72](https://github.com/yegor256/wring/issues/72) is now out of scope

**Comment by 0crat** [comment]
Order was finished, quality is "good": +35 point(s) just awarded to @paulodamaso[/z](https://www.0crat.com/u/paulodamaso)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0pdd** [comment]
@yegor256 3 puzzles [#83](https://github.com/yegor256/wring/issues/83), [#85](https://github.com/yegor256/wring/issues/85), [#86](https://github.com/yegor256/wring/issues/86) are still not solved; solved: [#76](https://github.com/yegor256/wring/issues/76), [#77](https://github.com/yegor256/wring/issues/77), [#78](https://github.com/yegor256/wring/issues/78), [#82](https://github.com/yegor256/wring/issues/82).

**Comment by 0pdd** [comment]
@yegor256 2 puzzles [#85](https://github.com/yegor256/wring/issues/85), [#86](https://github.com/yegor256/wring/issues/86) are still not solved; solved: [#76](https://github.com/yegor256/wring/issues/76), [#77](https://github.com/yegor256/wring/issues/77), [#78](https://github.com/yegor256/wring/issues/78), [#82](https://github.com/yegor256/wring/issues/82), [#83](https://github.com/yegor256/wring/issues/83).
